scripts/microbe_stage/biome.as

```diff
@@ -126,12 +124,25 @@ ObjectID createChunk(CellStageWorld@ world, uint chunkId,  Float3 pos)
 
     // Rigid Body
     auto rigidBody = world.Create_Physics(chunkEntity, position);
+
+    //chunk properties
+    if (chunk.damages > 0.0f || chunk.deleteOnTouch){
+        auto damager = world.Create_DamageOnTouchComponent(chunkEntity);
+        damager.setDamage(chunk.damages);
+        damager.setDeletes(chunk.deleteOnTouch);
+        auto body = rigidBody.CreatePhysicsBody(world.GetPhysicalWorld(),
+            world.GetPhysicalWorld().CreateSphere(radius),mass,
+        //damage
+        world.GetPhysicalMaterial("chunkDamageMaterial"));
```

Comment:
What does this do? Shouldn't the return value of this function be used somewhere???

Reply:
You must be tired,  that’s part of the create physics method. See coma on line before comment

Reply:
I'm not going to accept that I'm blind. That line should obviously be indented as it's part of expression on the previous line.

Reply:
I agree about that

Reply:
Okay i have fixed this :)